openqabot/approver.py

```diff
@@ -134,6 +134,18 @@ def _approvable(self, inc: IncReq) -> bool:
         # everything is green --> add incident to approve list
         return True
 
+    def mark_job_as_acceptable_for_incident(self, job_id: int, incident_number: int):
+        try:
+            patch(
+                "api/jobs/"
+                + str(job_id)
+                + "/remarks?text=acceptable_for&incident_number="
+                + str(incident_number),
+                headers=self.token,
+            )
+        except RequestError as e:
+            log.info("Unable to mark job %i as acceptable for incident %i", job_id, inc)
```

Comment:
Based on the CI failure https://github.com/openSUSE/qem-bot/actions/runs/13992943421/job/39180834093?pr=191#step:6:19

```suggestion
            log.info("Unable to mark job %i as acceptable for incident %i: %e", job_id, inc, e)
```